client/components/common/BaseForm.vue

```diff
@@ -38,6 +38,11 @@
       >
       <small v-if="hint"> {{ hint }} </small>
       <small v-if="errors[id]" class="error"> {{ errors[id] }} </small>
+      <small v-if="instructions" class="tooltip"> 
+        <img src = "question_circle.svg" alt=instructions width="40%"/>
+        <div class="tooltiptext" v-if="instructions">{{ instructions }}</div>
```

Comment:
I don't think `alt=instructions` works! Change to `:alt=instructions` or `:alt="instructions"` to access the actual data :)